packages/rocketchat-push-notifications/client/views/pushNotificationsFlexTab.js

```diff
@@ -88,7 +90,7 @@ Template.pushNotificationsFlexTab.onCreated(function() {
 		const field = this.editing.get();
 		const value = this.$('input[name='+ field +']:checked').val();
 		if (this.validateSetting(field)) {
-			Meteor.call('saveNotificationSettings', Session.get('openedRoom'), field, value, (err, result) => {
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: ["Meteor" is not defined.](https://www.codacy.com/app/RocketChat/Rocket-Chat/file/2313313532/issues/source?bid=3199634&fileBranchId=3199636#l93)